pinot-core/src/main/java/org/apache/pinot/core/data/readers/ParquetRecordReader.java

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
Can you download pinot codestyle file and apply it to this file? It's located in `config/codestyle-eclipse.xml` or `config/intellij.xml`.

Reply:
Thanks for your review. I'v reformatted all my changes.